app/hooks/usePeerConnection.tsx

```diff
@@ -1,8 +1,14 @@
-import { PartyTracks, type PartyTracksConfig } from 'partytracks/client'
+import {
+	PartyTracks,
+	setLogLevel,
+	type PartyTracksConfig,
+} from 'partytracks/client'
 import { useObservableAsValue } from 'partytracks/react'
 import { useEffect, useMemo, useState } from 'react'
 import { useStablePojo } from './useStablePojo'
 
+setLogLevel('debug')
```

Comment:
Is suppose to stay in here like that? :-)

Reply:
Yes!